test/recipe/admin/utils/giveMe.js

```diff
@@ -52,6 +52,25 @@ export default {
     return { key: keyAndTestId, title, component };
   },
 
+  anErrorPage({ safeLinks, errorResults, data}){
+    const errorInfos = {};
+    errorResults.forEach(element => {
+      errorInfos[element.statusCode] = { ...element };
+    });
```

Comment:
there should be child element builder, take `aHeaderItem` or `aSideMenuItem` for instance